modules/integration/tests-ui/src/test/resources/testng.xml

```diff
@@ -25,7 +25,7 @@
             <class name="org.wso2.carbon.appmanager.integration.ui.TestCases.AppDeleteTestCase"/>
             <class name="org.wso2.carbon.appmanager.integration.ui.TestCases.SelfSignUpTestCase"/>
             <class name="org.wso2.carbon.appmanager.integration.ui.TestCases.DocumentTestCase"/>
-            <class name="org.wso2.carbon.appmanager.integration.ui.TestCases.GoogleAnalyticsTestCase"/>
+            <!--<class name="org.wso2.carbon.appmanager.integration.ui.TestCases.GoogleAnalyticsTestCase"/>-->
```

Comment:
I AppM we use google Analytics for stat publishing and it will need google analytics account for this.